### ISSUE TITLE
Playlist resets when connectimg to Android Auto

### ISSUE THREAD

**Issue by ILuvSilicon** [VinylMusicPlayer/VinylMusicPlayer#1083]
**Describe the bug**
After I connect to Android Auto using a USB-C connector, all my playlists reset/empty. The playlists themselves aren't deleted, and their names stay unchanged, it only removes all the songs from the playlists.

Notable is: This doesn't occur if I have the app opened on my phone (currently use the app) while I connect. I usually get a notification "cannot play this song" when it happens. It twnda to happen when a song from a playlist was paused at the time i attempt to connect.

**To reproduce**
1. Play a song, then pause it
2. Connect to android auto using USB-C while at home screen (app in background)
3. Check if playlists were wiped

**Expected behavior**
When I connect to Android Auto, my playlists will not be wiped, and I will be given a chance to unpause the previous song, or choose a new song to play if not.

**Device info**
App version: 1.10.4
App version code: 201
Android build version: 33.0210.0210.332
Android release version: 13
Android SDK version: 33
Android build ID: TKQ1.220807.001.33.0210.0210.332 release-keys
Device brand: asus
Device manufacturer: asus
Device name: ASUS_I006D
Device model: ASUS_I006D
Device product name: EU_I006D
Device hardware name: qcom
ABIs: [arm64-v8a, armeabi-v7a, armeabi]
ABIs (32bit): [armeabi-v7a, armeabi]
ABIs (64bit): [arm64-v8a]

**Additional context**
I drive an electric Dacia spring if it helps

**Comment by secure-username** [comment]
When you say "it only removes all the songs from the playlists", do you mean it remove all of the songs from your queue?

I see that behavior too, in the last few releases. I don't know exactly when it started.

```
App version: 1.10.4-master_3fe10cd_240510-1323 CI
App version code: 201
Android build version: 0711a29c12
Android release version: 14
Android SDK version: 34
Android build ID: lineage_guacamole-userdebug 14 AP1A.240405.002.A1 0711a29c12
Device brand: OnePlus
Device manufacturer: OnePlus
Device name: OnePlus7Pro
Device model: GM1911
Device product name: OnePlus7Pro
Device hardware name: qcom
ABIs: [arm64-v8a, armeabi-v7a, armeabi]
ABIs (32bit): [armeabi-v7a, armeabi]
ABIs (64bit): [arm64-v8a]
```

**Comment by ILuvSilicon** [comment]
No, I mean it literally wipes clean all my playlists. I can have multiple playlists with however many songs in them, and then poof, all gone. The playlist will then contain 0 songs

![Screenshot_20240514-174701097 (1)](https://github.com/VinylMusicPlayer/VinylMusicPlayer/assets/81490933/fe02e281-000c-406f-853b-393102319648)

**Comment by peterjanes** [comment]
This happened to me yesterday, also tied to using Vinyl with Android Auto (over wifi though, not USB). Even the "automatic" lists were cleared, except for "Not recently played"; they've started populating again from scratch.

**Comment by geiseri** [comment]
I also see this on my pixel
```App version: 1.11.0
App version code: 202
Android build version: 11260668
Android release version: 14
Android SDK version: 34
Android build ID: UP1A.231105.001.B2
Device brand: google
Device manufacturer: Google
Device name: bramble
Device model: Pixel 4a (5G)
Device product name: bramble
Device hardware name: bramble
ABIs: [arm64-v8a, armeabi-v7a, armeabi]
ABIs (32bit): [armeabi-v7a, armeabi]
ABIs (64bit): [arm64-v8a]```

I don't have playlists but it clears any songs until I open the app on the phone and start playing something. 

It seems to work for a while then, but will stop when the application is quit.
